components/NavLayout/SideBar.js

```diff
@@ -32,6 +43,15 @@ export default function SideBar({ children, ...props }) {
     setOpen(false);
   };
 
+  const fetchUserInfo = () => {
+    getUserInfo('51').then(res => {
+      console.log(res)
+      setCurrentUserProfileInfo({
+        fullName: res.name.first_name + ' ' + res.name.last_name,
```

Comment:
Will need to add error handling for possible changes in API data. Handle later when we do the switch to dynamic routing, that way we can grab the uid from all pages. I'll also post an example of what error handling looks like in the components

Reply:
Will do!